Sources/NIOHTTP1/HTTPUpgradeHandler.swift

```diff
@@ -249,7 +249,6 @@ public class HTTPServerUpgradeHandler: ChannelInboundHandler {
             return ctx.eventLoop.makeSucceededFuture(result: ())
         }
 
-        return EventLoopFuture<Void>.andAll(self.extraHTTPHandlers.map { ctx.pipeline.remove(handler: $0).map { (_: Bool) in () }},
-                                            eventLoop: ctx.eventLoop)
+        return EventLoopFuture<Void>.andAll(self.extraHTTPHandlers.map { ctx.pipeline.remove(handler: $0).map { }}, eventLoop: ctx.eventLoop)
```

Comment:
I made this 1 line, but not sure what the cutoff length should be

Reply:
that `.map { }` can go. Hmm, I'd break the line after the opening brace for the first map